grafana-plugin/src/pages/integration/Integration.helper.ts

```diff
@@ -124,4 +125,5 @@ export const IntegrationHelper = {
   },
 };
 
-export const getIsBidirectionalIntegration = ({ integration }: AlertReceiveChannel) => integration === 'servicenow';
+export const getIsBidirectionalIntegration = ({ integration }: ApiSchemas['AlertReceiveChannel']) =>
+  integration === ('servicenow' as ApiSchemas['AlertReceiveChannel']['integration']); // TODO: add service now in backend schema as valid value and remove casting
```

Comment:
@vadimkerr seems that we need to support `servicenow` as a valid integration value in openapi schemas